packages/visualizations/addon/serializers/line-chart.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+import Ember from 'ember';
+import VisualizationSerializer from 'navi-visualizations/serializers/visualization';
+import { parseMetricName } from 'navi-data/utils/metric';
+
+const { isPresent, get } = Ember;
+
+export default VisualizationSerializer.extend({
+  /**
+   * Normalizes payload so that it can be applied to models correctly
+   * @param type - class type
+   * @param visualization - json parsed object
+   * @return {Object} normalized payload
+   */
+  normalize(type, visualization) {
+    if(visualization) {
+      let config = get(visualization, 'metadata.axis.y.series.config'),
+          seriesType = get(visualization, 'metadata.axis.y.series.type');
+
+      if(isPresent(config)) {
```

Comment:
Is there a case where `config` is undefined when `visualization` isn't?

Reply:
The default value for config in the line-chart model is `{}` so I have this check to handle the case where that default value is used.